client/shipping-label/views/purchase/steps/packages/add-item.js

```diff
@@ -81,7 +80,16 @@ const AddItemDialog = ( {
 					{ itemOptions }
 				</div>
 				<ActionButtons buttons={ [
-					{ label: __( 'Add' ), isPrimary: true, onClick: () => moveItem( sourcePackageId, movedItemIndex, openedPackageId ) },
+					{ label: __( 'Add' ), isPrimary: true, onClick: () => {
+						Object.keys( addedItems || {} ).forEach( ( sourcePackageId ) => {
+							Object.keys( addedItems[ sourcePackageId ] || {} ).forEach( ( movedItemIndex ) => {
+								if ( addedItems[ sourcePackageId ][ movedItemIndex ] ) {
+									moveItem( sourcePackageId, movedItemIndex, openedPackageId );
```

Comment:
What do you think about refactoring `moveItem()` to become `moveItems()` so we only dispatch one action?